It's needed to fix pylint errors first:

```
report_custom_filename/model/ir_actions_report_xml.py:50:17: E122 continuation line missing indentation or outdented
report_custom_filename/model/ir_actions_report_xml.py:51:17: E122 continuation line missing indentation or outdented
report_custom_filename/model/ir_actions_report_xml.py:52:17: E122 continuation line missing indentation or outdented
report_custom_filename/model/ir_actions_report_xml.py:53:13: E122 continuation line missing indentation or outdented
```

--- a/base_report_assembler/ir_report.py
+++ b/base_report_assembler/ir_report.py
@@ -94,8 +94,8 @@ class ReportAssembleXML(orm.Model):
         for rep in self.browse(cr, uid, ids, context=context):
             if rep.report_type != 'assemblage':
                 continue
-            if (vals.get('report_name', False)
-                    and vals['report_name'] != rep.report_name):
+            if (vals.get('report_name', False) and
+                    vals['report_name'] != rep.report_name):
                 report_name = vals['report_name']
             else:
                 report_name = rep.report_name

--- a/report_custom_filename/model/ir_actions_report_xml.py
+++ b/report_custom_filename/model/ir_actions_report_xml.py
@@ -47,8 +47,8 @@ class IrActionsReportXml(orm.Model):
                 cr, uid, context['active_ids'], context=context)
             return mako_template_env.from_string(
                 report.download_filename).render({
-                'objects': objects,
-                'o': objects[0],
-                'object': objects[0],
-            })
+                    'objects': objects,
+                    'o': objects[0],
+                    'object': objects[0],
+                })
         return False

